dawn/main/ansible/Ansible.js

```diff
@@ -16,12 +19,31 @@ const StudentCodeStatus = DawnData.StudentCodeStatus;
 
 const runtimeBuilder = ProtoBuf.loadProtoFile(`${protoFolder}/runtime.proto`);
 const RuntimeData = runtimeBuilder.build('RuntimeData');
+const TCPData = runtimeBuilder.build('TCPData');
+
+let test = 0;
```

Comment:
I agree that the use of the `test` variable here is largely unnecessary, why not just print what you receive for this test?